### PR TITLE
add gpt dist inference

### PR DESCRIPTION
**Pull request by wangxicoding**
### PR types

New features
### PR changes

Others
### Description

add gpt dist inference script

### CODE REVIEW

File: examples/language_model/gpt-3/deploy/python/inference.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+import os
+import tempfile
+from functools import partial
+
+import numpy as np
+import paddle
+import paddle.distributed.fleet as fleet
+from paddle import inference
+from paddlenlp.data import Stack, Tuple, Pad
+from paddlenlp.transformers import GPTForGreedyGeneration, GPTChineseTokenizer, GPTTokenizer
```
Comment (qingqing01) [inline examples/language_model/gpt-3/deploy/python/inference.py]: 1. remove importing in line 24: 「from paddlenlp.data import Stack, Tuple, Pad」
2. GPTForGreedyGeneration is not used

File: examples/language_model/gpt-3/deploy/python/inference.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+
+MODEL_CLASSES = {
+    "gpt-cn": (GPTForGreedyGeneration, GPTChineseTokenizer),
+    "gpt": (GPTForGreedyGeneration, GPTTokenizer),
```
Comment (qingqing01) [inline examples/language_model/gpt-3/deploy/python/inference.py]: GPTForGreedyGeneration is MODEL_CLASSES is not used

File: examples/language_model/gpt-3/deploy/python/inference.py (hunk 1 above)
Comment (wangxicoding) [reply examples/language_model/gpt-3/deploy/python/inference.py]: Done

File: examples/language_model/gpt-3/deploy/python/inference.py (hunk 2 above)
Comment (wangxicoding) [reply examples/language_model/gpt-3/deploy/python/inference.py]: Done

**Review by qingqing01 — APPROVED** [review]

**Review by ZHUI — APPROVED** [review]
LGTM

**Pull request merged**